### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scopeName' of undefined

### ISSUE THREAD

**Issue by gkvallis** [msiniscalchi/atom-latextools#52]
[Enter steps to reproduce below:]

1. This error arose when I opened the Changelog.md (preview) for latextools for version 0.8, after downloading version 0.8 from the master rather than going through the regular package update procedure. So it may not be a real error, as I have some kind of hybrid version on my Mac, but i report it anyway. 
2. ...

**Atom Version**: 1.5.4
**System**: Mac OS X 10.11.3
**Thrown From**: [latextools](https://github.com/msiniscalchi/atom-latextools) package, v0.7.7

### Stack Trace

Uncaught TypeError: Cannot read property 'scopeName' of undefined

```
At /Users/gkv/Dropbox/dotfiles/dot-atom/packages/latextools/lib/ltconsole.coffee:63

TypeError: Cannot read property 'scopeName' of undefined
    at /Users/gkv/Dropbox/dotfiles/dot-atom/packages/latextools/lib/ltconsole.coffee:63:52
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at stoppedChangingActivePaneItemCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:337:36)
```

### Commands

```
     -8:50.1.0 core:paste (atom-text-editor.editor.is-focused)
  5x -8:44.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -8:32.5.0 core:save (atom-text-editor.editor.is-focused.autocomplete-active)
     -8:28.3.0 core:backspace (atom-text-editor.editor.is-focused)
     -8:26.4.0 core:move-right (atom-text-editor.editor.is-focused)
     -8:25 core:backspace (atom-text-editor.editor.is-focused)
  2x -8:21.6.0 core:delete (atom-text-editor.editor.is-focused)
     -8:12.9.0 core:save (atom-text-editor.editor.is-focused)
  2x -8:04.0 core:backspace (atom-text-editor.editor.is-focused)
     -7:58.3.0 latextools:jump-to-pdf (atom-text-editor.editor.is-focused)
  3x -7:39.8.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -7:23.9.0 core:save (atom-text-editor.editor.is-focused)
     -6:58.5.0 latextools:build (atom-text-editor.editor.is-focused)
  2x -6:00.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -5:58.7.0 core:save (atom-text-editor.editor.is-focused)
     -5:09.9.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-glatex-theme.theme-atom-light-ui)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "linter-python-pyflakes",
      "wrap-guide",
      "vim-mode"
    ],
    "themes": [
      "atom-light-ui",
      "glatex-theme"
    ]
  },
  "latextools": {
    "builderSettings": {},
    "darwin": {},
    "hideBuildPanel": "no_errors",
    "linux": {},
    "win32": {}
  }
}
```

### Installed Packages

```coffee
# User
atom-html-preview, v0.1.16
autocomplete-python, v1.7.0
glatex-theme, v0.0.0
language-latex, v0.6.1
latex, v0.30.0
latextools, v0.7.7
linter, v1.11.3
linter-flake8, v1.13.0
linter-pyflakes, v0.3.0
markdown-preview-plus, v2.2.2
pdf-view, v0.48.0
terminal-plus, v0.14.5

# Dev
No dev packages
```

**Comment by ig0774** [comment]
msiniscalchi fixed this in ee481a8.

**Comment by msiniscalchi** [comment]
Indeed. The fix should be in 0.8.0. Can you update LaTeXTools and try again? If the problem is solved, please close.

**Comment by gkvallis** [comment]
Seems fixed, thanks.
